src/sentry/utils/json.py

```diff
@@ -40,6 +43,19 @@ def better_default_encoder(o):
     raise TypeError(repr(o) + ' is not JSON serializable')
 
 
+class JSONEncoderForHTML(_JSONEncoderForHTML):
```

Comment:
No, because I want to maintain this: https://github.com/simplejson/simplejson/blob/master/simplejson/encoder.py#L362-L378